modules/user/server-ts/sql.js

```diff
@@ -180,13 +188,15 @@ class User {
       .where({ id });
   }
 
-  async editUserProfile({ id, profile }) {
-    const userProfile = await knex
+  isUserProfile({ id }) {
```

Comment:
Why did you decided to take this function out and make separate? Also, the function name doesn't correlate with what it does.